docs/source/notebooks/finch-usage.ipynb

```diff
@@ -81,9 +101,9 @@
      "text": [
       "ProcessSucceeded\n",
       "frost_daysResponse(\n",
-      "    output_netcdf='https://pavics.ouranos.ca/wpsoutputs/e8afbb04-42d3-11ea-9531-0242ac12000b/frost-days_SRES-A2-experiment_20460101-20650101.nc',\n",
-      "    output_log='https://pavics.ouranos.ca/wpsoutputs/e8afbb04-42d3-11ea-9531-0242ac12000b/log.txt',\n",
-      "    ref='https://pavics.ouranos.ca/wpsoutputs/e8afbb04-42d3-11ea-9531-0242ac12000b/input.meta4'\n",
+      "    output_netcdf='http://localhost:5000/outputs/13b909e4-686e-11ea-a94b-b052162515fb/frost-days_SRES-A2-experiment_20460101-20650101.nc',\n",
```

Comment:
Revert all output url from `localhost` to `pavics.ouranos.ca`?  With the new regex, leaving it as `localhost` will pass Jenkins but given this notebooks is used as tutorial, it's weird the output url do not match the host.

Your call, for me what's most important is Jenkins is not broken.

Reply:
But how do I update the notebook before the working server is live at pavics.ouranos.ca ?

Reply:
> But how do I update the notebook before the working server is live at pavics.ouranos.ca ?

Manual, just for that cell ;(  Yeah it's ugly.  You can leave it as `localhost` as well, won't break Jenkins.  Users running the tutorials notebook might not notice as well.